src/site/content/en/blog/smoothness/index.md

```diff
@@ -0,0 +1,563 @@
+   properties.
+- Adjusting <strong style="font-weight:700;color:#78A55A">paint</strong>
+   properties.
+- Adjusting <strong style="font-weight:700;color:#78A55A">composite</strong>
+   properties.
+
+Because these stages are sequential, it is important to define animations in
+terms of properties further down this pipeline. The earlier the update, the
+greater the costs and the less likely it is to be smooth. (See [rendering
+performance](https://developers.google.com/web/fundamentals/performance/rendering)
+for more details.)
+
+So, while it can sometimes be convenient to animate layout properties, there are
+costs to doing so—even if not immediately apparent. Animations should, wherever
+possible, be defined in terms of composite property changes.
+
+Defining [declarative CSS animations or using Web
+Animations](https://developers.google.com/web/fundamentals/design-and-ux/animations/css-vs-javascript),
+and ensuring you [animate composite
+properties](https://developer.chrome.com/blog/hardware-accelerated-animations/),
+is a great initial step to ensuring smooth and efficient animations. But still,
+this alone does not guarantee smoothness because even efficient web animations
+have performance limits. That's why it is always important to measure!
+
+## What are animation frames?
+
+Updates to the visual representation of a page take time to appear. Visual
+changes will eventually lead to a new animation frame, which is eventually
+presented to the user via their display.
+
+Generally, displays can only update at certain intervals, so visual updates are
+batched. Many displays update on a fixed interval of time, such as 60 times a
+second (that is 60&nbsp;Hz). Some more modern displays can offer higher refresh rates
+(90-120&nbsp;Hz are becoming common). Often these displays can actively adapt between
+refresh rates as needed, or even offer fully variable frame rates.
+
+The goal for any application, like a game or a browser, is to process all these
+batched visual updates and produce a visually complete animation frame within
+the deadline, every time. Note that this goal is entirely distinct from other
+important browser tasks such as loading content from the network quickly or
+executing JavaScript tasks efficiently.
+
+At some point, it can become too difficult to complete all visual updates within
+the allotted deadline assigned by the display. When this happens, the browser
+**drops a frame**. Your screen doesn't go black, it just repeats itself. You see
+the same visual update for a bit longer—the same animation frame that was
+presented at the previous frame opportunity.
+
+This actually happens often! It is not necessarily even perceptible, especially
+for static or document-like content, which is common on the web platform in
+particular. Dropped frames only become apparent when there are important visual
+updates, such as animations, for which we need a steady stream of animation
+updates to show smooth motion.
```

Comment:
```suggestion
This can happen often, and isn't always perceptible, especially
for static or document-like content, which is common on the web platform.
Dropped frames only become apparent when there are important visual
updates, such as animations, for which we need a steady stream of updates to show smooth motion.
```